string new_string
---
string string_change_status